exercises/ex1.rb

```diff
@@ -0,0 +1,30 @@
+class Person
+  def initialize(name, age)
+    @name = name
+    @age = age
+  end
+
+  def display
+    puts "-name : #{@name}"
+    puts "-age : #{@age}"
+  end
+end
+
+p = Person.new("Tomas Wild", 37)
```

Comment:
đối với string không nội suy thì recommend dùng single-quote nhé
```suggestion
p = Person.new('Tomas Wild', 37)
```